### ISSUE TITLE
Do not assume applications to reside in /usr/bin

### ISSUE THREAD

**Issue by ayekat** [ayekat/dotfiles#10]
Currently, all wrapper scripts assume the executable's "real" location to be `/usr/bin`.

This assumption fails in the following cases:

* If the system administrator has installed the application to a different place (e.g. `/usr/local/bin`).
* If the system is not Arch Linux (i.e. `/usr/sbin` also exists).
* If the system has not undergone the `/usr`-merge yet (e.g. Debian).

A proposed solution would be to somehow obtain a "list" of valid paths to executables with that name, and strip the wrapper script itself from that list in order to have it execute the next entry in that list. I'm not sure if this is feasible, though.

**Comment by ayekat** [comment]
This could work:

```sh
# ... do stuff here

PATH="$(echo ":$PATH:" | sed -e "s|:$(dirname "$0"):|:|;s|^:||;s|:\$||")"
exec "$(basename "$0")"
```

~~Since the last two lines are going to be in every wrapper script, defining it in a separate file and sourcing it would be probably the most elegant.~~ At least the `qutebrowser` script is linked to from an external source, so it should be self-contained. For the other scripts, this remains to be seen (although, I can also live with copy-pasting two lines).

**Comment by ayekat** [comment]
I've added `path.sh`, since using `PATH` (even without exporting) affects the exec'd process, which would then not see the dotfiles' path anymore (see commit 65c200d).

For the wrapper scripts, I consider this issue to be solved. The systemd service files fall into the category "fix it when it breaks" (for Debian), and they can't be made location-agnostic in regards to `/usr/local` anyway, since they always require a full path.

tl;dr Closing this.
